### PR TITLE
[Feature] Deconstructing a Cloning Pod removes the Biomass from it as Synthmeat slabs

### PR DESCRIPTION
**Pull request by DeityLink**
As was requested. You can also remove the biomass without prying out the components since the meat is spawned right when you use the crowbar. (of course the biomass is removed at the same time the meat is spawned, so no exploit)

### CODE REVIEW

**Comment by 9600bauds** [comment]
Good Yes

**Comment by ComicIronic** [comment]
Could you name the issue for the autocloser?

**Comment by DeityLink** [comment]
I put the issue Num in the changelog's commit.

**Comment by clusterfack** [comment]
You DID realize that the cloner starts with 150 biomass right? Deconstruct construct deconsruct construct deconstruct construct to make infinite biomass.

Make a map cloner version that starts with 150 and make cloners start with 0 if you are going to do this.

**Comment by DeityLink** [comment]
>cloner starts with 150 biomass

oh god why? wait, in that case can't they just deconstruct-reconstruct it between each cloning? sounds like an already existing exploit.

**Comment by clusterfack** [comment]
It pretty much is

File: code/modules/medical/cloning.dm [hunk 1]
```diff
@@ -30,6 +30,9 @@
 
 	l_color = "#7BF9FF"
 
+/obj/machinery/cloning/clonepod/empty
```
Comment (ComicIronic) [inline code/modules/medical/cloning.dm]: I don't think this is the right order to go about it.
The map version should start with biomass, but the machine itself should start with nothing.

Comment (DeityLink) [inline code/modules/medical/cloning.dm]: Dude I'm not turning this PR into a map PR, how many of these do we have open already?

Comment (ComicIronic) [inline code/modules/medical/cloning.dm]: We have 3 valid map PRs open. 2 are either varedits or path replacements (nothing to do with cloning). The third doesn't edit any tiles with clonepods on them.

Comment (d3athrow) [inline code/modules/medical/cloning.dm]: make a map child of clonepod, use NP++ to change the types on the maps and you're done.

**Comment by DeityLink** [comment]
sigh...<font size='4'>fine</font>

**Pull request merged**